File: docs/.validation/csharp/getting-started_5.cs
// Source: getting-started.md:1251
using GitHub.Copilot.SDK;

using var client = new CopilotClient(new CopilotClientOptions
{
    CliUrl = "localhost:4321",
    UseStdio = false
});

// Use the client normally
await using var session = await client.CreateSessionAsync();
// ...

File: docs/.validation/python/getting-started_5.py
# Source: getting-started.md:1210
import asyncio

async def main():
    from copilot import CopilotClient
    
    client = CopilotClient({
        "cli_url": "localhost:4321"
    })
    await client.start()
    
    # Use the client normally
    session = await client.create_session()
    # ...

asyncio.run(main())